orangecanvas/scheme/readwrite.py

```diff
@@ -804,16 +1006,16 @@ def check(obj):
             return all(map(check, obj))
```

Comment:
@ales-erjavec, on Friday I didn't yet realize that .ows files contain literals, not json. I am always a few (well, a many) steps behind. :|

The list of allowed sequences does not include sets. I suppose it's because there's no literal for an empty set. But could non-empty sets be allowed?

Reply:
No. It would need to be all sets or none at all.